fenix/messaging.fml.yaml

```diff
@@ -156,7 +156,6 @@ features:
               style: NOTIFICATION
               trigger:
                 - I_AM_NOT_DEFAULT_BROWSER
-                - DAY_3_AFTER_INSTALL
```

Comment:
To be re-added an investigate as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1821274

Reply:
I'd leave this for now, given that it is a default. If we need to add something that triggers immediately for everyone, then leave this here, but if not, move the whole thing into a `developer` channel.